storybookjs/storybook
# remove confusing drop shadow from info body sections

Issue: https://github.com/storybooks/storybook/issues/1877

Drop shadow creates strange look in Info inline mode

## What I did

Removed it

## How to test

Check Netlify

Is this testable with jest or storyshots?
Y, screenshots may need to be updated
Does this need a new example in the kitchen sink apps?
N
Does this need an update to the documentation?
N
If your answer is yes to any of these, please make sure to include it in your PR.
